README.md

```diff
@@ -399,6 +399,7 @@ All contributions are welcome just send a pull request.  Jackett's framework all
 
 ### Windows
 * Open the Jackett solution in Visual Studio 2017
+* Install all missing assemblies (Jackett, Jackett.Console, Jackett.Service etc.)
```

Comment:
Can we change this to 'Right click on the Jackett solution and click 'Rebuild Solution' to restore nuget packages'